go.mod

```diff
@@ -4,6 +4,7 @@ go 1.12
 
 replace (
 	github.com/go-resty/resty => gopkg.in/resty.v1 v1.9.0
+	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
```

Comment:
I had to add this because upstream moved packages which caused the following error on `go get`: 

```
build github.com/rancher/types: cannot load knative.dev/pkg/apis/istio: module knative.dev/pkg@latest found (v0.0.0-20191210184447-d5d6ec46971d), but does not contain package knative.dev/pkg/apis/istio
```